tools/README.md

```diff
@@ -36,9 +36,22 @@ The jar is generated in below directory :
 
 `./tools/target/rapids-4-spark-tools_2.12-<version>.jar`
 
+## How to use this tool
```

Comment:
We have `How to use this tool` section in both qualification and profiling sections below but it doesn't have examples of possible inputs.  Would it make sense to remove redundancy. Either keep this here and remove from qualification and profiling sections OR  remove here and move `possible inputs`  to both qualification and profiling.

Reply:
Yes, the `how to use this tool` headline can be renamed IMO for this section. May be `Overview of tools` or something similar?

Reply:
@tgravescs what do you think?

Reply:
I specifically move the  How to use this tool sections below each one because they are 2 different tools.

Are those sections not clear?

Reply:
I think, I updated an older version of the doc before your changes got applied. I will remove this change.